core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11294,4 +11294,49 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
         isLinux(expectedBigQuery));
   }
 
+
+  @Test public void testNextDayFunctionWithDate() {
+    final RelBuilder builder = relBuilder();
+    final RexNode nextDayRexNode = builder.call(SqlLibraryOperators.NEXT_DAY,
+        builder.literal("2023-02-22"), builder.literal("TUESDAY"));
+    final RelNode root = builder
+        .scan("EMP")
+        .project(builder.alias(nextDayRexNode, "next_day"))
+        .build();
+    final String expectedSpark =
+        "SELECT NEXT_DAY('2023-02-22', 'TUESDAY') next_day\n"
```

Comment:
Plz use existing constant or create new for the day (TUESDAY). Also add one more test case with other day.

Reply:
Done